doc/user/PARSING.md

```diff
@@ -0,0 +1,58 @@
+# Parsing options
+
+When setting options from the [CLI Options](OPTIONS.md), the [commands](COMMANDS.md) or using the [libf3d options string API](../libf3d/OPTIONS.md#string-api), the values are parsed according to their type.
+
+The following types are supported:
+ - bool
+ - int
+ - double
+ - ratio
+ - string
+
+As well as vectors for each of these types.
+
+## Bool
+
+The following formats are supported when parsing a bool, case insensitive:
+ - true/false
+ - yes/no
+ - on/off
+ - 1/0
+
+When formatting a bool into a string, true/false is used.
+
+## Int
+
+Int parsing is supported using [std::stoi](https://en.cppreference.com/w/cpp/string/basic_string/stol) and check
+that the whole string is parsed.
```

Comment:
What happens if the check fails? Document

Reply:
the value is not changed as parsing fails, I should document this behavior more generally though

Reply:
I\ve added a note about parsing failure but I cant really be more precise, as the exact behavior depends if the user uses the libf3d or F3D